CRM/Streetimport/GP/Handler/TEDIContactRecordHandler.php

```diff
@@ -359,8 +360,32 @@ public function performContactBaseUpdates($contact_id, $record) {
     $this->updateContact($contact_params, $contact_id, $record);
 
     //Address Entity
-    $address_params = $this->getPreparedAddressParams($record);
-    $this->createOrUpdateAddress($contact_id, $address_params, $record);
+    if ($this->addressChangeRecordedSince($contact_id, $this->getDate($record), $record)) {
```

Comment:
Similar to `PostalReturn_Base`, we want to use the Action activity date (see `getActivityId()`) as the reference date here - that's the time at which the address was exported; `getDate()` would return the call date included in the file instead (which could be days to months later).

(Thinking about this a bit more, the activity's `created_date` would actually be the most accurate, since we often set `activity_date_time` to something slightly different. Could change that for `PostalReturn_Base` too while we're at it.)